ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_activations.py

```diff
@@ -226,6 +226,34 @@ def test_silu(*, dtype_and_x, test_flags, backend_fw, fn_name, on_device):
     )
 
 
+# softshrink
+@handle_test(
+    fn_tree="functional.ivy.experimental.softshrink",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        large_abs_safety_factor=8,
+        small_abs_safety_factor=8,
+        safety_factor_scale="log",
+    ),
+    threshold=st.one_of(
+        st.floats(min_value=0.0, max_value=10.0),
```

Comment:
Why would you add limits?

Reply:
In the original implementation of torch.nn.softshrink. it's takes the values of lambd > 0.0. so i put it to min value =0.0. and if you suggest then i just removed the max value. bcz it don't have the constrain on the max value.

Reply:
Oh, I just noticed that I didn't check on that issue. What would happen if you removed `max_value`? Can you experiment on that locally?

I would be more comfortable if you removed the constraint, but if it gives an error you may put it as `1e15 or something large that doesn't give an overflow error

Reply:
I tested locally it's fine upto 1e30 max_value for 125 max test examples. i added this in test to max_value = 1e30